src/Workflow.py

```
@@ -410,6 +414,260 @@ def execution(self) -> None:
+                    f"{'secondary' if secondary else 'primary'})"
+                )
+            self.logger.log('Ignoring these files!')
+
+        self.logger.log('Listening for new raw files...')
+        while(True):
+            # Scan every 1s
+            sleep(1)
+            
+            # Search for new raws
+            new_raws, new_methods, new_secs = self._find_raws(raw_folder_path)
+            for raw, method, secondary in zip(new_raws, new_methods, new_secs):
```

Ruff (0.11.9)

Loop control variable `method` not used within loop body

(B007)